Site_Files/src/main/webapp/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+
+    <div class="header">
+        <a href="#default" class="logo">Resource Site</a>
+        <div class="header-right">
+            <a class="active" href="#home">Home</a>
+            <a href="#contact">Contact</a>
+            <a href="#login-signup">Login/Signup</a>
```

Comment:
We can change this to be two different links where a different one can appear depending on who is signed in later